week_2/pet.rb

```diff
@@ -46,7 +46,11 @@ def initialize(name: , animal_type_id: , food_consumed_per_day:)
 
   # Return the habitat of the pet
   def habitat
-    raise NotImplementedError # TODO
+    for key in HABITATS.keys
+      if HABITATS[key].include? animal_type_id
+        return key
+      end
+    end
```

Comment:
You can make use of [`find`](https://apidock.com/ruby/Enumerable/find) method here.

Reply:
I have made the changes